### PR TITLE
Unescape the less than and greater than inequalities

### PR DESCRIPTION
**Pull request by peterjc**
This change seems sensible for both reading the input markdown, and the rendering as HTML.

Assuming there is not something in the template which deals with this (and my change is not needed), then I would guess this fix will be needed on all the other recently migrated lessons (and the migration script may need tweaking).

Update: Looking at a few of the other lessons with R and Python (by eye), I did not find any more unescaped inequalities.

### CODE REVIEW

**Comment by gvwilson** [comment]
I haven't had a chance to review - could you please?

**Comment by abostroem** [comment]
@gvwilson not sure how to test this. When I run 
make preview 
it only updates the README.html...

**Comment by peterjc** [comment]
You can look at the README.md file in your favourite markdown aware viewer, or just browse the files as of my commit right here on GitHub using GitHub's rendering, e.g.:
https://github.com/peterjc/python-novice-inflammation/blob/a4009e55195018f3a1e5b04e2fcc7f3b278170de/01-numpy.md

**Comment by aaren** [comment]
@peterjc you missed one:

```
$ ack '&#' -G .*md
01-numpy.md
39:numpy.loadtxt(fname=&#39;inflammation-01.csv&#39;, delimiter=&#39;,&#39;)
```

**Comment by aaren** [comment]
@abostroem you can force make to update everything by doing `make preview -B`

**Comment by aaren** [comment]
@gvwilson any idea why this is like this in the first place?

**Comment by gvwilson** [comment]
@aaren Apologies for losing the thread - why what is like what?

**Comment by peterjc** [comment]
@gvwilson I think the question was how did these over-escaped HTML entities get into the markdown source?

@aaren Good catch on the HTML escaped quotes, give my a moment and I'll fix that in another commit...

**Comment by aaren** [comment]
@gvwilson yes, why are these html things (`&#39;`, etc.) in the markdown at all? Is there a systemic reason?

**Comment by peterjc** [comment]
Rebased and the quote fixed.

**Comment by peterjc** [comment]
I haven't down a grep across all the lessons (I only have a handful cloned locally), but it looks like something went wrong on this lesson only?

**Comment by aaren** [comment]
Here is the full (?) list of html entities still in the markdown:

```
$ ack '&' -G '.*md$'
05-defensive.md
71:<ipython-input-19-33d87ea29ae4> in <module&gt;()

04-cond.md
412:    That means that on a 40&times;60 data set, we are performing the same calculation 2400 times.

01-numpy.md
39:numpy.loadtxt(fname=&#39;inflammation-01.csv&#39;, delimiter=&#39;,&#39;)
238:if we have an M&times;N array in Python,
468:The expression `(40,)` tells us we have an N&times;1 vector,

```

@peterjc can you fix the defensive programming one as well?

As for the others (`&times`), I'm not sure what to do with these. If we had mathjax we could replace with latex, but I don't think that we do so either leave them as is or replace with `x`.

**Comment by peterjc** [comment]
I thought I did all the less-than and greater-than signs, thanks again @aaren.

I'm not sure what is best for the times signs - they should look nice in the HTML output at least.

**Comment by gvwilson** [comment]
It's almost certainly a side effect of an automatic translator (probably 
the IPython Notebook to Markdown translation).  There's no reason to 
keep the escapes.  I'm happy to replace them all with "natural" characters.

File: 01-numpy.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
           <h1 class="title">Programming with Python</h1>
           <h2 class="subtitle">Analyzing Patient Data</h2>
 <div id="learning-objectives" class="objectives objectives">
-<h2>Learning Objectives</h2>
+<h2 id="learning-objectives" class="objectives objectives">Learning Objectives</h2>
```
Comment (aaren) [inline 01-numpy.html]: Does this need to change with this PR? I know it is happening automatically, but could you move it to a different commit?

File: 05-defensive.html [hunk 2]
```diff
@@ -28,7 +28,7 @@
           <h1 class="title">Programming with Python</h1>
           <h2 class="subtitle">Defensive Programming</h2>
 <div id="learning-objectives" class="objectives objectives">
-<h2>Learning Objectives</h2>
+<h2 id="learning-objectives" class="objectives objectives">Learning Objectives</h2>
```
Comment (aaren) [inline 05-defensive.html]: also here.

**Comment by abostroem** [comment]
@aaren and @peterjc does the current form render correctly and this just fixes the code to be all html or does the current version (pre-PR) render incorrectly?

Please let me know when you are ready to merge

**Comment by aaren** [comment]
@abostroem the pre-PR version renders incorrectly. Look at `01-numpy.html` in your browser. This PR modifies the markdown input to not use html escaped characters and then updates the html to reflect that.

**Comment by peterjc** [comment]
I deliberately committed in the changes to ``*.html`` as generated by ``make preview`` as per my understanding of the instructions.

(Personally I don't understand why generated ``*.html`` files are under source code control, but that's a different issue.)

**Comment by gvwilson** [comment]
> I deliberately committed in the changes to |*.html| as generated by 
> |make preview| as per my understanding of the instructions.
>

Thanks.

> (Personally I don't understand why generated |*.html| files are under 
> source code control, but that's a different issue.)
>
From https://github.com/swcarpentry/lesson-template/blob/gh-pages/DESIGN.md:

> Using Pandoc instead of Jekyll means that we have to compile our 
> Markdown into HTML on our own machines and commit it to the |gh-pages| 
> branch of the lesson's GitHub repository.
>
> It's considered bad practice to put computer-generated files under 
> version control, but the HTML pages put into the lesson's root 
> directory by Pandoc /must/ be committed to version control in order 
> for the lesson to be displayed properly on GitHub.
>
> It's also considered bad practice to put generated files in the same 
> directory as source files. We do it because some authors may use 
> auxiliary source files such as R Markdown and IPython Notebook files 
> which are executable, and contain paths to things like images and data 
> files. If we put our source files in a sub-directory, those paths 
> won't work (or conversely, if we put our generated files in a 
> different directory, the paths won't work there).
>

-- 
Dr. Greg Wilson    | gvwilson@software-carpentry.org
Software Carpentry | http://software-carpentry.org

**Comment by aaren** [comment]
This looks good to me. I'll leave it to @abostroem to merge.

**Pull request merged**